volume/fs_test.go

```diff
@@ -0,0 +1,135 @@
+package volume
+
+import (
+	"context"
+	"testing"
+
+	"github.com/puppetlabs/wash/datastore"
+	"github.com/puppetlabs/wash/plugin"
+	"github.com/stretchr/testify/mock"
+	"github.com/stretchr/testify/suite"
+)
+
+const varLogFixture = `
```

Comment:
A brief comment indicating that this fixture represents the output of `StatCmd(/var/log)` would help.